core/src/main/java/org/frankframework/http/rest/MediaTypes.java

```diff
@@ -40,7 +41,56 @@ public enum MediaTypes {
 	OCTET("application/octet-stream", null), // raw binary formats do not have a charset
 	MULTIPART_RELATED("multipart/related"),
 	MULTIPART_FORMDATA("multipart/form-data"),
-	MULTIPART("multipart/*");
+	MULTIPART("multipart/*"),
+
+	// less but still common types
+	TXT("text/plain"),
+	CSS("text/css"),
+	CSV("text/csv"),
+	MJS("text/javascript"),
+	HTML("text/html"),
+	JAR("application/java-archive"),
+
+	DOC("application/msword"),
+	EPUB("application/epub+zip"),
```

Comment:
Ik snap dat je dit heb gebaseerd op een lijstje (zag ik in je comment), maar zijn ebooks echt relevant voor ons framework?

Reply:
Haha, nee, maar dit zijn de meestgebruikte web-mimetypes. Mocht iemand zo gek zijn op deze manier een website te genereren dan zou dat nu kunnen?